deploy/template.yaml

```diff
@@ -448,14 +446,17 @@ Resources:
             - Name: NODE_ENV
               Value: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, nodeEnv ]
             - Name: RUM_APPLICATION_ID
-              Value:
-                Fn::ImportValue: !Sub "AppMonitorId-${Environment}"
+              Value: !If [ IsNotDevelopment,
+                  Fn::ImportValue: !Sub "AppMonitorId-${Environment}",
+                  !Ref AWS::NoValue ]
             - Name: RUM_ROLE_ARN
-              Value:
-                Fn::ImportValue: !Sub "CognitoRoleARN-${Environment}"
+              Value: !If [ IsNotDevelopment,
+                  Fn::ImportValue: !Sub "CognitoRoleARN-${Environment}",
+                  !Ref AWS::NoValue ]
```

Comment:
Presumably CognitoRoleARN-{env} and CognitoPoolID-{env} are also non-dev exports?

Reply:
yes